src/components/Card.jsx

```diff
@@ -1,52 +1,31 @@
 
 import { useState } from 'react'
 import SingleCard from './SingleCard';
-import { data } from "../assets/bioData.js";
+// import { data } from "../assets/BioData.js";
 
```

Comment:
Rather than importing data here, I imported it in `App.jsx` and passed it as a prop down to this component